packages/@webex/webex-core/src/lib/services-v2/interceptors/service.js

```
@@ -0,0 +1,101 @@
+const trailingSlashes = /(?:^\/)|(?:\/$)/;
+
+/**
+ * @class
+ */
+export default class ServiceInterceptor extends Interceptor {
+  /**
+   * @returns {ServiceInterceptor}
+   */
+  static create() {
+    /* eslint no-invalid-this: [0] */
+    return new ServiceInterceptor({webex: this});
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)